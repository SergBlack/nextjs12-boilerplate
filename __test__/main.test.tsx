import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Home from '@/pages/index';

describe('Home', () => {
  it('should to match snapshot', () => {
    const { container } = render(<Home />);

    expect(container).toMatchSnapshot();
  });
});
