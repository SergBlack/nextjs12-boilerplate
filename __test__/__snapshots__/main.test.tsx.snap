// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should to match snapshot 1`] = `
<div>
  <div
    style="margin-top: 100px;"
  >
    <div
      class="text-center mb-5"
    >
      <p
        class="mb-0 mt-3 text-disabled"
      >
        Welcome to the world !
      </p>
    </div>
    <div>
      <form
        class="ant-form ant-form-vertical"
      >
        <div
          class="ant-row ant-form-item"
        >
          <div
            class="ant-col ant-col-8 ant-form-item-label"
          >
            <label
              class=""
              title="Input Number"
            >
              Input Number
            </label>
          </div>
          <div
            class="ant-col ant-col-8 ant-form-item-control"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <div
                  class="ant-input-number ant-input-number-lg"
                  style="width: 100px;"
                >
                  <div
                    class="ant-input-number-handler-wrap"
                  >
                    <span
                      aria-disabled="false"
                      aria-label="Increase Value"
                      class="ant-input-number-handler ant-input-number-handler-up"
                      role="button"
                      unselectable="on"
                    >
                      <span
                        aria-label="up"
                        class="anticon anticon-up ant-input-number-handler-up-inner"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="up"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M890.5 755.3L537.9 269.2c-12.8-17.6-39-17.6-51.7 0L133.5 755.3A8 8 0 00140 768h75c5.1 0 9.9-2.5 12.9-6.6L512 369.8l284.1 391.6c3 4.1 7.8 6.6 12.9 6.6h75c6.5 0 10.3-7.4 6.5-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                    <span
                      aria-disabled="false"
                      aria-label="Decrease Value"
                      class="ant-input-number-handler ant-input-number-handler-down"
                      role="button"
                      unselectable="on"
                    >
                      <span
                        aria-label="down"
                        class="anticon anticon-down ant-input-number-handler-down-inner"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="down"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                  </div>
                  <div
                    class="ant-input-number-input-wrap"
                  >
                    <input
                      aria-valuemax="10"
                      aria-valuemin="1"
                      aria-valuenow="3"
                      autocomplete="off"
                      class="ant-input-number-input"
                      name="inputNumber"
                      role="spinbutton"
                      step="1"
                      value="3"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-row ant-form-item"
        >
          <div
            class="ant-col ant-col-8 ant-form-item-label"
          >
            <label
              class=""
              title="Switch"
            >
              Switch
            </label>
          </div>
          <div
            class="ant-col ant-col-8 ant-form-item-control"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <button
                  aria-checked="true"
                  class="ant-switch ant-switch-checked"
                  role="switch"
                  type="button"
                >
                  <div
                    class="ant-switch-handle"
                  />
                  <span
                    class="ant-switch-inner"
                  />
                </button>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-row ant-form-item"
        >
          <div
            class="ant-col ant-col-8 ant-form-item-label"
          >
            <label
              class=""
              title="Slider"
            >
              Slider
            </label>
          </div>
          <div
            class="ant-col ant-col-8 ant-form-item-control"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <div
                  class="ant-slider ant-slider-horizontal"
                >
                  <div
                    class="ant-slider-rail"
                  />
                  <div
                    class="ant-slider-track"
                    style="left: 0%; width: 70%;"
                  />
                  <div
                    class="ant-slider-step"
                  />
                  <div
                    aria-disabled="false"
                    aria-valuemax="100"
                    aria-valuemin="0"
                    aria-valuenow="70"
                    class="ant-slider-handle"
                    role="slider"
                    style="left: 70%; transform: translateX(-50%);"
                    tabindex="0"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-row ant-form-item"
        >
          <div
            class="ant-col ant-col-8 ant-form-item-label"
          >
            <label
              class=""
              title="Select"
            >
              Select
            </label>
          </div>
          <div
            class="ant-col ant-col-8 ant-form-item-control"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <div
                  class="ant-select ant-select-lg ant-select-single ant-select-show-arrow"
                  style="width: 192px;"
                >
                  <div
                    class="ant-select-selector"
                  >
                    <span
                      class="ant-select-selection-search"
                    >
                      <input
                        aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
                        aria-autocomplete="list"
                        aria-controls="rc_select_TEST_OR_SSR_list"
                        aria-haspopup="listbox"
                        aria-owns="rc_select_TEST_OR_SSR_list"
                        autocomplete="off"
                        class="ant-select-selection-search-input"
                        id="rc_select_TEST_OR_SSR"
                        readonly=""
                        role="combobox"
                        style="opacity: 0;"
                        type="search"
                        unselectable="on"
                        value=""
                      />
                    </span>
                    <span
                      class="ant-select-selection-item"
                      title="lucy"
                    >
                      lucy
                    </span>
                  </div>
                  <span
                    aria-hidden="true"
                    class="ant-select-arrow"
                    style="user-select: none;"
                    unselectable="on"
                  >
                    <span
                      aria-label="down"
                      class="anticon anticon-down ant-select-suffix"
                      role="img"
                    >
                      <svg
                        aria-hidden="true"
                        data-icon="down"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="64 64 896 896"
                        width="1em"
                      >
                        <path
                          d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                        />
                      </svg>
                    </span>
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-row ant-form-item"
          style="margin-top: 48px;"
        >
          <div
            class="ant-col ant-col-8 ant-col-offset-8 ant-form-item-control"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <button
                  class="ant-btn ant-btn-primary ant-btn-lg"
                  type="submit"
                >
                  <span>
                    OK
                  </span>
                </button>
                <button
                  class="ant-btn ant-btn-default ant-btn-lg"
                  style="margin-left: 8px;"
                  type="button"
                >
                  <span>
                    Cancel
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
      </form>
    </div>
  </div>
</div>
`;
